docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
 \subsection{Probleme während der Durchführung}
 \colorbox{red}{TODO:}
 
+
+
+
 \section{Projekt Meilensteine}
+
 \subsection{Meilenstein 1}
-\colorbox{red}{TODO:}Kurze Einleitung, von wann bis wann ging
+Der erste Meilenstein begann am 29.10.2020 und ging bis zum 11.11.2020. Für den Anfang des Projektes war hier die Planung, Struktur und das Vorgehen des Projektes der Fokus.
```

Comment:
```suggestion
Die Laufzeit des ersten Meilensteins erstreckte sich vom 29.10.2020 bis zum 11.11.2020 und war auf
die Planung, Struktur und das Vorgehen des Projektes fokusiert.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee